src/proto/script_host.proto

```diff
@@ -122,7 +125,14 @@ message TwoFactorResponse {
     string two_factor = 2;
 }
 
-// Next: 8
+// Next: 2
+// Client message to update the local changes to the repository
+message NewChanges {
+    // A Git patch file containing the update
+    bytes patch = 1;
+}
```

Comment:
This seems like the same thing as a patch in the GitFS sense. Is that right? If so, could this script proto and the gitfs proto share a patch message?

Reply:
Yeah, we can do that. I'd still like to keep a `NewChanges` message here, as that makes it easier to change the structure of this command in the future.

Reply:
Without `NewChanges`, it wouldn't be any harder to change than `ProjectSpec`, right? It feels inconsistent to me. Should we add another level of indirection to `ProjectSpec`?

Reply:
Look at ProjectSpec - I did.

Reply:
The levels in ProjectSpec are ProjectSpec -> Patch -> bytes
The levels in SessionClientMessage are SessionClientMessage -> NewChanges -> Patch -> bytes

My point is: can we have SessionClientMessage -> Patch -> bytes instead?

Reply:
Ah. Yes, I suppose that works. It just means that if we ever want to add something to that command, we need to go by it a different route.